cli/common/common.go

```diff
@@ -1,7 +1,12 @@
 package common
```

Comment:
A `common` package has the potential to become a dumping ground for random stuff. Maybe we can keep this in `//cli/version` but move `HandleVersion` to a separate `//cli/versioncmd` package to break the circular dep?